openshift_cli_installer/tests/cluster_version/test_osd_rosa_versions.py

```diff
@@ -0,0 +1,103 @@
+import click
+import pytest
+
+from openshift_cli_installer.tests.cluster_version.rosa_osd_base_versions import ROSA_OSD_BASE_VERSIONS
+from openshift_cli_installer.utils.cluster_versions import (
+    get_cluster_version_to_install,
+)
+
+PARAMETRIZE_TESTS = [
+    ([{"version": "4.15", "stream": "stable", "expected": "4.15.8"}]),
```

Comment:
we should add a check in user input that IP get the right streams and rosa etc get the right channel-group